subprojects/launcher/src/integTest/groovy/org/gradle/launcher/continuous/SmokeContinuousIntegrationTest.groovy

```diff
@@ -55,10 +56,17 @@ class SmokeContinuousIntegrationTest extends AbstractContinuousIntegrationTest {
         waitBeforeModification(markerFile)
         markerFile.text = "created"
         then:
-        buildTriggeredAndSucceeded()
-        output.contains "exists: true"
+        if (OperatingSystem.current().linux) {
+            buildTriggeredAndSucceeded()
+            output.contains "exists: true"
+        } else {
+            // TODO: We may want to support this use-case for
+            //       hierarchical watchers at some point as well.
+            noBuildTriggered()
+        }
```

Comment:
I think it would be good to have some more comments here describing why this only works on Linux now.

Reply:
I added some more comments.